stl.go

```diff
@@ -782,6 +857,7 @@ func (s Subtitles) WriteToSTL(o io.Writer) (err error) {
 			return
 		}
 	}
+
```

Comment:
I'm being picky here, but can you not add this line feed? It messes up the git diff

Reply:
Done